EIPS/eip-FOCIL.md

```diff
@@ -100,11 +100,11 @@ We make the following changes to the engine API:
 
 The full consensus changes can be found in the following Github repository. They are split between: 
 
-- [Beacon Chain](https://github.com/terencechain/consensus-specs/blob/ae2cf0e1285a0ca64dd339fb84551d4af20280e6/specs/_features/focil/beacon-chain.md) changes.
-- [Fork choice](https://github.com/terencechain/consensus-specs/blob/ae2cf0e1285a0ca64dd339fb84551d4af20280e6/specs/_features/focil/fork-choice.md) changes.
-- [P2P](https://github.com/terencechain/consensus-specs/blob/ae2cf0e1285a0ca64dd339fb84551d4af20280e6/specs/_features/focil/p2p-interface.md) changes.
-- [Honest validator guide](https://github.com/terencechain/consensus-specs/blob/ae2cf0e1285a0ca64dd339fb84551d4af20280e6/specs/_features/focil/validator.md) changes.
-- [Fork logic](https://github.com/terencechain/consensus-specs/blob/ae2cf0e1285a0ca64dd339fb84551d4af20280e6/specs/_features/focil/fork.md) changes.
+- [Beacon Chain](https://github.com/terencechain/consensus-specs/blob/6056b69ea1215c3dff6042da2b0a8563347be645/specs/_features/focil/beacon-chain.md) changes.
+- [Fork choice](https://github.com/terencechain/consensus-specs/blob/6056b69ea1215c3dff6042da2b0a8563347be645/specs/_features/focil/fork-choice.md) changes.
+- [P2P](https://github.com/terencechain/consensus-specs/blob/6056b69ea1215c3dff6042da2b0a8563347be645/specs/_features/focil/p2p-interface.md) changes.
+- [Honest validator guide](https://github.com/terencechain/consensus-specs/blob/6056b69ea1215c3dff6042da2b0a8563347be645/specs/_features/focil/validator.md) changes.
+- [Fork logic](https://github.com/terencechain/consensus-specs/blob/6056b69ea1215c3dff6042da2b0a8563347be645/specs/_features/focil/fork.md) changes.
```

Comment:
This leads to 404.

Reply:
sorry I think that should have been engine api